go.mod

```diff
@@ -37,37 +37,37 @@ require (
 	gopkg.in/yaml.v3 v3.0.1
 	gotest.tools v2.2.0+incompatible
 	gotest.tools/v3 v3.4.0
-	k8s.io/api v0.27.2
-	k8s.io/apiextensions-apiserver v0.27.2
-	k8s.io/apimachinery v0.27.2
-	k8s.io/apiserver v0.27.2
-	k8s.io/cli-runtime v0.27.2
-	k8s.io/client-go v0.27.2
-	k8s.io/component-helpers v0.27.2
+	k8s.io/api v0.28.0
+	k8s.io/apiextensions-apiserver v0.28.0
+	k8s.io/apimachinery v0.28.0
+	k8s.io/apiserver v0.28.0
+	k8s.io/cli-runtime v0.28.0
+	k8s.io/client-go v0.28.0
+	k8s.io/component-helpers v0.28.0
```

Comment:
I just checked the lib versions, and there's already a `v0.28.1`.

It might make sense to update to `v0.28.1` here.

Reply:
I was just trying to place safe and update to the bare minimum. :wink: Will try that out now.